docs/controller-skills/towerstrips.md

```diff
@@ -0,0 +1,117 @@
+
+    This is to prevent incidental placement onto the runway.
+
+#### Control Bar 
+
+The bottom of the window holds the Control Bar. This panel contains the server connection status, aerodrome selected and ATIS code among other elements.
+
+The *INHIBIT* and *FOR STP* button respectively remove and add selected aircraft from the strip board as required. 
+
+The *XX CROSS XX* adds a red highlight to the strip, marking its intentions to cross an active runway, rather than depart from it.
+
+The *PDC* button opens the default vatSys PDC editor. 
+
+!!! Note
+    **Currently, PDCs through OzStrips can only be sent via Private Message, not through Hoppies.**
+    
+    To send a PDC through vatAcars, you'll be required to dispatch it manually.
+
+### Strips
+
+The background colour of the strip corresponds to its status as an arrival or a departure. Departing aircraft have a blue strip, while arrivals have a yellow strip.
+
+<figure markdown>
+![Strip](../controller-skills/img/ozstripstrip.png){ width="700" }
+</figure>
+
+| Number | Content | Action |
+|---------|-------------|--------------|
+| 1 | Bay Number | Change |
+| 2 | Filed Off Blocks Time | Cock Strip |
+| 3 | Aircraft Type | Open Flightplan |
+| 4 | Destination | Open Flightplan |
+| 5 | Route Indicator | Show Route |
+| 6 | Flight Rules | |
+| 7 | Correct SSR Code + Mode C Received | |
+| 8 | SSR Code | Autogenerate Code |
+| 9 | Wake Turbulence Category | |
+| 10 | Callsign | Select Strip |
+| 11 | Runway | Change |
+| 12 | Holding Point / Clearance Limit | Change |
+| 13 | SID | Move strip to next bay |
+| 14 | First Waypoint | Open flightplan |
+| 15 | Departure Heading | Change |
+| 16 | Requested Level | Open flightplan |
+| 17 | Cleared Level | Change |
+| 18 | Takeoff Timer | Start / Reset |
+| 19 | Global Ops Data | Change |
+| 20 | Local Remarks | Change |
+
+### Queueing and Reordering
+
+A strip can be placed into the queue by selecting the strip and then pressing "Add to Queue". Strips can be moved vertically by pressing the keyboard arrow keys while a strip is selected.
+
+## Workload Management
+
+While use of OzStrips can aid situational awareness, it is important to use it in such a manner that it doesn't negatively impact your productivity.
+
+### Cocking Strips
+
+Left clicking on the strip's EOBT will highlight and cock the strip, making it stand out. It is recommended to mark strips this way, if there is an outstanding request.
+
+### Queue Management
+
+When multiple aircraft have outstanding requests, it is recommended you 'queue up' the strip, by placing it below the queue bar. This can be done quickly by selecting the strip then pressing "Add to Queue".
+
+When an aircraft is in a queue, make sure the aircraft's bay location represents its current state. 
+
+!!! Example
+    An aircraft in a queue for an airways clearance, would remain in the *Preactive* bay, below the queue bar, not in the *Cleared* bay.
+
+### HMI
```

Comment:
Maybe this could be moved to be a `!!! tip` markdown item in Workload Management, as it's just a little reminder about good process, and we don't really use the acronym HMI in this environment :)